MacPass/MPPasswordInputController.m

```diff
@@ -138,6 +159,204 @@ - (IBAction)_submit:(id)sender {
   }
 }
 
+- (void) _touchIdHandleUnlockAttempt: (KPKCompositeKey*)compositeKey withResult:(bool)success {
+  if(success && self.databaseFileURL && self.databaseFileURL.lastPathComponent) {
```

Comment:
Can we rename this to something reflecting the actual purpose of storing the key and simply only call it if we have success == YES? I find it hard to read a function that gets skipped when passing a certain flag.

And if we retrieve the datbaseFileURL via the document I am unable to come up with a scenario where the URL might be nil. I might happen if the file gets pulled under our feet but then I would rely on ```[touchIDSecuredPasswords setObject:encryptedKey forKey:self.databaseFileURL.lastPathComponent]``` having a ```nil``` check around it.